pyiron_workflow/api.py

```diff
@@ -0,0 +1,19 @@
+"""
+Developer-focused API.
+
+Public items included here are subject to semantic versioning.
+"""
+
+# Node developer entry points
+from pyiron_workflow.channels import NOT_DATA
+from pyiron_workflow.find import (
+    find_nodes as _find_nodes,  # Not formally in API -- don't rely on interface
```

Comment:
Since the intention of `api.py` is to make a separate interface to developers, this name change is maybe not necessary?

Reply:
The change here is to indicate that it's not protected by semantic versioning, so it's still valid. [Both api's are otherwise protected.](https://github.com/pyiron/decisions/blob/1ff6601b45e22a9e97483123a5c2146a1704b9a6/decisions/25-001-split-api.md?plain=1#L18)